src/sagemaker/pipeline.py

```diff
@@ -104,6 +113,56 @@ def deploy(self, initial_instance_count, instance_type, endpoint_name=None, tags
         if self.predictor_cls:
             return self.predictor_cls(self.endpoint_name, self.sagemaker_session)
 
+    def _create_sagemaker_pipeline_model(self, instance_type):
+        """Create a SageMaker Model Entity
+
+        Args:
+            instance_type (str): The EC2 instance type that this Model will be used for, this is only
+                used to determine if the image needs GPU support or not.
+            accelerator_type (str): <put docs here>
```

Comment:
nit: Please remove this before this variable is added to the constructor.

Reply:
Oh dear. Will do.